pkg/runtime/scheduler/streamer.go

```diff
@@ -144,14 +145,29 @@ func (s *streamer) invokeApp(ctx context.Context, job *schedulerv1pb.WatchJobsRe
 		return fmt.Errorf("error returned from app channel while sending triggered job to app: %w", err)
 	}
 
-	statusCode := response.Status().GetCode()
-	switch codes.Code(statusCode) {
-	case codes.OK:
-		log.Debugf("Sent job: %s to app", job.GetName())
-	case codes.NotFound:
-		log.Errorf("non-retriable error returned from app while processing triggered job %v: %s. status code returned: %v", job.GetName(), statusCode)
+	switch s.isHTTP {
+	case true:
```

Comment:
Please use `if s.isHTTP { ... } else {`. `switch` is a bit awkward for booleans.

Reply:
q: why are we making a distinction for app protocol at all?

Reply:
We need the distinction to acct for http 200 throwing the unexpected error. I showed this to you before when you had your refactor PR open. I'll try to see if I have it saved

Reply:
I thought it was cleaner with the switch

Reply:
this: 
```
for scheduler, to an http app at trigger time Im seeing:
ERRO[0237] unexpected status code returned from app while processing triggered job test1. status code returned: 200` 
```

Reply:
I did update it to use an if statement instead of the switch per your request

Reply:
I don't like the fact that we need to make the distinction here, and makes the point of having an `app channel` abstraction a bit pointless if we always need to do the check anyway. Can we standardize on the error code returned by both protocol channels, converting HTTP status codes to gRPC codes?

Nit on the code in that function- the `if` statement checking for the 200 range should be at the top :sweat_smile: 
https://github.com/dapr/dapr/blob/master/pkg/messaging/v1/util.go#L299

Reply:
Can we do the conversion in a follow up PR to dapr/dapr? I can add an issue in dapr/dapr for it to not forget and tag it as a good first issue.

Reply:
I agree on the nit re: the 200 should go at the top

Reply:
Please can we add a TODO in the code?

Reply:
I added the code todo and moved the 200 check up